### DISCUSSION TITLE
Material difference in the top of the board border

### DISCUSSION THREAD

**Discussion by akaKarthick** [czbar/ChessForge#720] [Ideas]
I'm writing to you today to suggest a new feature for Chessforge: showing the material difference in the top of the board border. This would be a great way to quickly and easily see who is ahead in material, without having to count up all of the pieces on the board.

I think this feature would be especially useful for beginners like my niece, who may not be able to quickly and accurately assess the material balance on the board. It would also be useful for intermediate players like me, who want to be able to quickly see how their material balance is changing over the course of the game.

Lichess and Chess.com have different approaches to showing the material difference. Lichess eliminates the common pieces for both sides, while Chess.com shows all of the pieces. I think the Lichess approach is more accurate and informative, as it shows the true difference in material between the two players.

What are your thoughts on this feature? Do you think it would be useful for Chessforge users?

**Comment by czbar** [comment]
Well, if it would be useful for you and your niece then that answers the question, doesn't it :)  

I must admit that I barely even noticed those hints on chess.com/lichess/ChessBase. I was a beginner long before the computer chess era and quickly learned to "subconsciously" keep track of the material. Hence, it didn't occur to me that it might be actually useful.  But if it is, I will schedule it for implementation once the current bout of refactorings is over.
